### PR TITLE
[ergoCubSN000] Add wbd - FTs IMU and yarpmotorgui configurations

### PR DESCRIPTION
**Pull request by lrapetti**
Hi, we open this PR with the configuration on which we have been working on in https://github.com/ami-iit.

Credits to @GiulioRomualdi @mebbaid

### CODE REVIEW

File: ergoCubSN000/hardware/inertials/left_arm-eb2-j0_1-inertial.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    <group name="PROTOCOL">
+                        <param name="major">            0                   </param>
+                        <param name="minor">            0                   </param>
+                    </group>
+
+                    <group name="FIRMWARE">
+                        <param name="major">            0                   </param>
+                        <param name="minor">            0                   </param>
+                        <param name="build">            0                   </param>
+                    </group>
```
Comment (Nicogene) [inline ergoCubSN000/hardware/inertials/left_arm-eb2-j0_1-inertial.xml:28]: These two versions set to 0 are fine @marcoaccame @sgiraz @davidetome ?

**Review by Nicogene — COMMENTED** [review]
I have only a doubt about the protocol and firmware version for the strain2 inertial service, but it seems ok to me 👍🏻

File: ergoCubSN000/hardware/inertials/left_arm-eb2-j0_1-inertial.xml (hunk 1 above)
Comment (sgiraz) [reply ergoCubSN000/hardware/inertials/left_arm-eb2-j0_1-inertial.xml:28]: Hi @Nicogene,

Actually, `strain2` and `strain2c` have the following configuration:

```cpp
constexpr embot::app::theCANboardInfo::applicationInfo applInfo
{
    embot::prot::can::versionOfAPPLICATION {2, 0, 15},
    embot::prot::can::versionOfCANPROTOCOL {2, 0}
};
```

**Review by sgiraz — APPROVED** [review]
Ok for me!

File: ergoCubSN000/hardware/inertials/left_arm-eb2-j0_1-inertial.xml (hunk 1 above)
Comment (pattacini) [reply ergoCubSN000/hardware/inertials/left_arm-eb2-j0_1-inertial.xml:28]: Thank you @sgiraz 👍🏻 
@lrapetti could you update the application and protocol versions accordingly?

File: ergoCubSN000/hardware/inertials/left_leg-eb9-j4_5-inertial.xml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+                    <group name="PROTOCOL">
+                        <param name="major">            0                   </param>
+                        <param name="minor">            0                   </param>
+                    </group>
+
+                    <group name="FIRMWARE">
+                        <param name="major">            0                   </param>
+                        <param name="minor">            0                   </param>
+                        <param name="build">            0                   </param>
```
Comment (pattacini) [inline ergoCubSN000/hardware/inertials/left_leg-eb9-j4_5-inertial.xml:27]: Same here.

File: ergoCubSN000/hardware/inertials/right_arm-eb1-j0_1-inertial.xml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+                    <group name="PROTOCOL">
+                        <param name="major">            0                   </param>
+                        <param name="minor">            0                   </param>
+                    </group>
+
+                    <group name="FIRMWARE">
+                        <param name="major">            0                   </param>
+                        <param name="minor">            0                   </param>
+                        <param name="build">            0                   </param>
```
Comment (pattacini) [inline ergoCubSN000/hardware/inertials/right_arm-eb1-j0_1-inertial.xml:27]: Same here.

File: ergoCubSN000/hardware/inertials/right_leg-eb7-j4_5-inertial.xml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+                    <group name="PROTOCOL">
+                        <param name="major">            0                   </param>
+                        <param name="minor">            0                   </param>
+                    </group>
+
+                    <group name="FIRMWARE">
+                        <param name="major">            0                   </param>
+                        <param name="minor">            0                   </param>
+                        <param name="build">            0                   </param>
```
Comment (pattacini) [inline ergoCubSN000/hardware/inertials/right_leg-eb7-j4_5-inertial.xml:27]: Same here.

**Review by pattacini — CHANGES_REQUESTED** [review]
Thanks @lrapetti for the additions 👍🏻 
Could you only take some minutes to update the version numbers as suggested by @sgiraz?

**Comment by pattacini** [comment]
> Thanks @lrapetti for the additions 👍🏻 Could you only take some minutes to update the version numbers as suggested by @sgiraz?

Hold on!
It seems that for `strain2` we've been gone with 0.0.0. See:
- https://github.com/robotology/robots-configuration/blob/devel/iCubGenova09/hardware/inertials/left_leg-eb7-IMU.xml#L16-L29

So, let's go this way now. We can refine later on.

**Review by pattacini — APPROVED** [review]

**Pull request merged**